src/ingest-pipeline/misc/tools/airflow_environments/env_stage.sh

```diff
@@ -2,7 +2,8 @@
 HM_AF_CONFIG=/opt/repositories/vm003-stage/ingest-pipeline/src/ingest-pipeline/airflow/airflow.cfg
 HM_AF_HOME=/opt/repositories/vm003-stage/ingest-pipeline/src/ingest-pipeline/airflow
 
-HM_AF_CONN_INGEST_API_CONNECTION=http://vm003.hive.psc.edu:7777/
+# Use DEV since STAGE does not have INGEST API
+HM_AF_CONN_INGEST_API_CONNECTION=http://vm001.hive.psc.edu:7777/
```

Comment:
wasn't vm003 the stage vm?

Reply:
It is, but the STAGE environment doesn't have ingest API deployed on it - so I pointed it at DEV instead